user_account/api/views.py

```diff
@@ -1,11 +1,49 @@
-from rest_framework import generics, status
-from rest_framework.decorators import api_view
+import jwt
+from django.conf import settings
+from django.shortcuts import get_object_or_404
+from django.views.decorators.csrf import ensure_csrf_cookie
+from rest_framework import exceptions, generics, status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 
-from user_account.api.serializers import RegistrationSerializer
-from user_account.models.user import UserAccount
+from rest_framework_jwt.serializers import jwt_payload_handler
+from user_account.api.serializers import UserAccountSerializer
+from user_account.models.user_account import UserAccount
 
 
 class UserCreateView(generics.CreateAPIView):
-    serializer_class = RegistrationSerializer
+    serializer_class = UserAccountSerializer
     queryset = UserAccount.objects.all()
+    permission_classes = [AllowAny]
+
+
+class UserLoginView(generics.CreateAPIView):
+    serializer_class = UserAccountSerializer
+    queryset = UserAccount.objects.all()
+    permission_classes = [AllowAny]
+
+    def create(self, request):
+        email = request.data.get("email")
+        password = request.data.get("password")
+
+        user = UserAccount.objects.get(email=email)
```

Comment:
UserAccount.objects.get(email=email) can raise exception if you don't have Account with that email